pasqal_cloud/client.py

```diff
@@ -108,18 +110,27 @@ def _request(
         payload: Optional[Union[Mapping, Sequence[Mapping]]] = None,
         params: Optional[Mapping[str, Any]] = None,
     ) -> JSendPayload:
-        rsp = requests.request(
-            method,
-            url,
-            json=payload,
-            timeout=TIMEOUT,
-            headers={"content-type": "application/json"},
-            auth=self.authenticator,
-            params=params,
-        )
-        rsp.raise_for_status()
-        data: JSendPayload = rsp.json()
-
+        successful_request: bool = False
```

Comment:
There is currently no override, I don't know if we want this to be forced default behavior (you would not see it take place if no retry happens) or if we want to make it configurable with an argument. I'd argue for the override arg but can't think of a scenario you want to

Reply:
Suggestion: We could make the number of retries an optional argument of this function. This would both give us the ability to control the number of retries or disable retries altogether by setting it to 0. 

This is not required for now, just a thought